dot/build_spec_integration_test.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//go:build integration
+// +build integration
+
+package dot
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/genesis"
+	"github.com/ChainSafe/gossamer/lib/runtime"
+	"github.com/stretchr/testify/require"
+)
+
+const codeHex = "0x3a636f6465"
```

Comment:
Maybe add a comment to what this is? 🤔 Is it `code` in hex?

Reply:
Good point, added comment.